dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch

```diff
@@ -0,0 +1,29 @@
+https://bugs.gentoo.org/936060
+--- a/Drivers/Postgre7.1/info.c
```

Comment:
Could you email this to upstream please? (nick.gorham@easysoft.com)

I'll merge once that's done.

Reply:
Upstream commit https://github.com/lurcher/unixODBC/commit/45f501e1be2db6b017cc242c79bfb9de32b332a1

Reply:
Ah, then it should've been in the patch description to begin with ;)

Could you add that to the top of the patch please? Thanks.

Reply:
I wasn't too familiar with what could be done in patch files. I've pushed a new description text now. Thanks for the feedback :)

Reply:
Thank you! See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions for future reference as well